test/cbus-netid.test.js

```diff
@@ -198,19 +198,14 @@ test('construct illegal whitespace', function (assert) {
 });
 
 test('construct illegal project name', function (assert) {
-	assert.plan(4);
+	assert.plan(3);
 
 	assert.throws(function () {
 		// non numerical group
 		CBusNetId.parse(`//S,3AC/254/56`);
 	});
 
 	assert.throws(function () {
-		// first char not alpha
```

Comment:
rather than removing test, probably better to change this to test that the name is now allowed -- perhaps there's a more appropriate test function for it...

while you are there, pls add in a test for names including an underscore.